editor-components/logo-clouds/logo-comp2/logo-comp2.module.scss

```diff
@@ -1,316 +1,62 @@
-                align-items: center;
-                margin: 0 auto;
-
-                .title {
-                    margin: 0;
-                    text-align: center;
-                    font-size: 2.5rem;
-                    text-transform: uppercase;
-                    color: var(--composer-primary-color);
-
-                    &.row {
-                        width: 30%;
-
-                        @media screen and (max-width: 768px) {
-                            display: flex;
-                            align-items: center;
-                            justify-content: center;
-                        }
-                    }
-                }
-
-                .description {
-                    width: 100%;
-                    margin: 40px 0;
-                    text-align: center;
-                    text-transform: uppercase;
-                    color: var(--composer-font-color-primary);
-
-                    &.row {
-                        width: 70%;
-                    }
-                }
-
+    
+            &:hover{
+                animation-play-state: paused;
             }
-
-            section {
-                &.row {
-                    width: 100%;
-                    display: flex;
-                    flex-wrap: nowrap;
-
-                    @media screen and (max-width: 768px) {
-                        flex-direction: column;
-                        display: block;
-                        align-items: center;
-                    }
-
-
-                }
-            }
-
-            .items-layout {
-                display: flex;
-                flex-direction: row;
-                flex-wrap: wrap;
-                justify-content: center;
-                gap: 20px;
-                padding: 0 20px;
-
-
-                .all-images {
-                    display: flex;
-                    align-items: center;
-                    justify-content: center;
-                    // background: red;
-
-                    
-
-                    .image-child {
-                        display: flex;
-                        width: 100%;
-                        justify-content: center;
-                        flex-wrap: wrap;
-                        align-items: center;
-                        flex-direction: row;
-                        gap: 20px;
-                        float: left;
-                        
-                        
-
-                        img {
-                            // margin-bottom: 20px;
-                            width: 100%;
-                            height: 100%;
-
-                        }
-
-                    }
+            .image-child{
+                display: inline-block;
+                padding: var(--composer-gap-md) var(--composer-gap-lg);
+                background: color-mix(in srgb, var(--composer-html-background), var(--composer-font-color-primary) 5%);
+                border-radius: 200px;
```

Comment:
We need to use css varible for border radius

Reply:
There is not enough border-radius variable for this view. The maximum border-radius is less than 200px. For these cases, it is okay to send a specific border radius as in the example.